### DISCUSSION TITLE
Consistent naming

### DISCUSSION THREAD

**Discussion by scoreorder** [musescore/MuseScore#21605] [General]
The items of the dropdown box of the Family menu of the Choose Instruments window are named sometimes after music genres and sometimes after music ensembles. I suggest naming the items of the dropdown box consistently after music genres.

Common
All
Pop / Rock
Jazz
Orchestral Music
Choral Music
Wind Music
March Music
Electronic Music
World Music
Early Music
School Music

**Comment by dcorson-ticino-com** [comment]
I'm puzzled, where would I find String Quartet in that list.  Nothing is obvious.
Chamber Music seems to be missing.
